cli/polka-storage/storagext/src/runtime/client.rs

```diff
@@ -41,30 +47,18 @@ impl Client {
     ) -> Result<Self, subxt::Error> {
         let rpc_address = rpc_address.as_ref();
 
-        let mut current_retries = 0;
-        loop {
-            let client = if cfg!(feature = "insecure_url") {
```

Comment:
I think this was the only place where we used `insecure_url` feature. Maybe we can also remove it entirely

Reply:
Totally forgot about that, agreed